packages/player-component/index.js

```diff
@@ -586,6 +586,8 @@ class Player extends Nanocomponent {
       b.setAttribute('title', playing ? 'Pause' : 'Play')
       morph(i, icon(playing ? 'pause' : 'play', { class: i.classList }))
```

Comment:
This is where the problem is. An update will be ready this week

Reply:
I see. Thank you for your comment.